app_dart/lib/src/model/appengine/branch.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:gcloud/db.dart';
+import 'package:github/github.dart';
+
+@Kind(name: 'Branch', idType: IdType.String)
```

Comment:
Do we need json serialization support?

Reply:
umm I am not sure if we are going to need it, but I added one in case we need it in future

Reply:
Yes, we will. We'll want to send this to the frontend.